openbas-model/src/main/java/io/openbas/helper/InjectModelHelper.java

```diff
@@ -28,47 +32,89 @@ public static boolean isReady(
       @NotNull final List<String> teams,
       @NotNull final List<String> assets,
       @NotNull final List<String> assetGroups) {
-    if (injectorContract == null) {
+    if (injectorContract == null || content == null) {
       return false;
     }
-    if (content == null) {
-      return false;
+
+    ObjectMapper mapper = new ObjectMapper();
+    ArrayNode injectContractFields;
+
+    try {
+      injectContractFields =
+          (ArrayNode)
+              mapper
+                  .readValue(injectorContract.getContent(), ObjectNode.class)
+                  .get(CONTACT_CONTENT_FIELDS);
+    } catch (JsonProcessingException e) {
+      throw new RuntimeException("Error parsing injector contract content", e);
```

Comment:
Would that really be a runtime exception ? The given JSON is client geven, so wouldn't it be more a badrequest ?

Reply:
It was simpler for me to use a runtime rather than having to propagate the exception in the methods above.
But I'm open to suggestions.

Reply:
Let's go with RTE for now and we will see if we need to change it in the future, then :)